spec/features/workflow/unwithdraw_without_managing_editor_permissions_spec.rb

```diff
@@ -23,7 +23,7 @@ def and_i_click_on_undo_withdrawal
 
   def then_i_see_a_message_to_ask_my_managing_editor_to_unwithdraw_content
     expect(page).to have_content(
-      I18n.t!("unwithdraw.no_managing_editor_permission.title"),
+      I18n.t!("unwithdraw.no_manage_live_history_mode_permission.title", title: @withdrawn_edition.title),
```

Comment:
It doesn't feel quite right that this test needs to be changed - maybe I've missed something?

Reply:
You're correct, reverted back!